src/maintainer/conda_forge_yml.rst

```diff
@@ -378,6 +392,8 @@ To enable ``linux_ppc64le`` and ``linux_aarch64`` and the following:
       linux_ppc64le: default
       linux_aarch64: default
 
+If the ``build_platform`` for an arch is not available with the selected provider, the build will be disabled; cross-compilation must be specified manually.
```

Comment:
I'm actually not sure if the build will be disabled or if the build will be created but just probably fail.

Reply:
We emulate aarch64.

Reply:
What default means is actually set in smithy itself and subject to change.